Features3/tests/features/editContact.feature

```diff
@@ -0,0 +1,40 @@
+Feature: Edit contact
```

Comment:
for this feature
make scenarios 1 as : make a single user and edit its data
make scenarios 2 as: make multiple user added to contact and edit a specific user data

these two would be enough